src/Orchard.Web/Modules/Orchard.Autoroute/Migrations.cs

```diff
@@ -49,5 +62,46 @@ public class Migrations : DataMigrationImpl {
 
             return 4;
         }
+
+        public int UpdateFrom4() {
+            // Adding some culture neutral patterns if they don't exist
+            var types = _contentManager.GetContentTypeDefinitions().Where(t => t.Parts.Any(p => p.PartDefinition.Name.Equals(typeof(AutoroutePart).Name)));
```

Comment:
I am sure you can even get the ContentTypePartDefinition with AutoroutePart for all type